src/App.tsx

```diff
@@ -1,9 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { FC } from 'react';
+
+interface Movie {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+export const App: FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setQuery(value);
+  };
+
+  const visibleMovies: Movie[] = moviesFromServer.filter(
+    ({ title, description }: Movie) => {
+      const keyWord = query.toLowerCase().trim();
+      const titleIncludes = title.toLowerCase().includes(keyWord);
+      const descrIncludes = description.toLowerCase().includes(keyWord);
```

Comment:
you share the same logic - toLowerCase and includes between title and description comparison, create helper func for it